### PR TITLE
workflow para gradle toma1

### PR DESCRIPTION
**Pull request by Antonieta13**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
```
Comment (aperdomob) [inline .github/workflows/gradle.yml]: remove this line and give execution permission in your repo

File: build.gradle [hunk 2]
```diff
@@ -14,6 +14,11 @@ dependencies {
     testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
     implementation group: 'org.seleniumhq.selenium', name: 'selenium-java', version: '3.141.59'
     testImplementation("io.github.bonigarcia:webdrivermanager:4.4.3")
+    implementation group: 'org.seleniumhq.selenium', name: 'selenium-firefox-driver', version: '3.141.59'
+
+
+
+
```
Comment (aperdomob) [inline build.gradle]: remove lines

File: src/test/java/GoogleSearch.java [hunk 3]
```diff
@@ -15,9 +17,16 @@
...
-        driver = new FirefoxDriver();
+         WebDriverManager.firefoxdriver().setup();
+
+        FirefoxBinary firefoxBinary = new FirefoxBinary();
+        FirefoxOptions options = new FirefoxOptions();
+        options.setBinary(firefoxBinary);
+        options.setHeadless(true);
+        driver = new FirefoxDriver(options);
+
+
```
Comment (aperdomob) [inline src/test/java/GoogleSearch.java]: remove lines

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: src/test/java/GoogleSearch.java (hunk 3 above)
Comment (Antonieta13) [reply src/test/java/GoogleSearch.java]: Done

File: build.gradle (hunk 2 above)
Comment (Antonieta13) [reply build.gradle]: Done

File: .github/workflows/gradle.yml (hunk 1 above)
Comment (Antonieta13) [reply .github/workflows/gradle.yml]: Done

File: gradlew [hunk 4]
```diff
@@ -14,7 +14,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
+#
```
Comment (aperdomob) [inline gradlew]: this change is not needed

**Review by aperdomob — CHANGES_REQUESTED** [review]

File: gradlew (hunk 4 above)
Comment (Antonieta13) [reply gradlew]: Done

**Review by aperdomob — APPROVED** [review]

**Pull request merged**